PowerEditor/src/MISC/Common/Sorters.h

```diff
@@ -130,6 +130,49 @@ class LexicographicSorter : public ISorter
 	}
 };
 
+// Implementation of lexicographic sorting of lines, ignoring character casing
+class LexicographicCaseInsensitiveSorter : public ISorter
+{
+public:
+	LexicographicCaseInsensitiveSorter(bool isDescending, size_t fromColumn, size_t toColumn) : ISorter(isDescending, fromColumn, toColumn) { };
+
+	std::vector<generic_string> sort(std::vector<generic_string> lines) override
+	{
+		// Note that both branches here are equivalent in the sense that they give always give the same answer.
```

Comment:
"they give always give" seems to be one give to much

Reply:
If you search the code for those four words BEFORE this PR, you will find them.
Thus, this code is blameless for that, as it is a copy+paste. :-)
But, I will fix both.